### ISSUE TITLE
Calling /api/v1/me/prefs with no field specified yields HTTP 500

### ISSUE THREAD

**Issue by thatJavaNerd** [reddit/reddit#1509]
If I'm not mistaken there seems to be a bug in `/api/v1/me/prefs`. Requesting specific preferences works fine, but trying to request them all (specifying no query parameters) returns a 500 Internal Server Error.

With `?fields=over18`:

```sh
$ curl -H 'Authorization: bearer <token>' \
    -H 'User-Agent: desktop:net.dean.jraw.test:v0.8.0' \
    "https://oauth.reddit.com/api/v1/me/prefs?fields=over_18"
{"over_18": true}
```

Without specifying `fields`:

```sh
$ curl  -H 'Authorization: bearer <token>' \
    -H 'User-Agent: desktop:net.dean.jraw.test:v0.8.0' \
    "https://oauth.reddit.com/api/v1/me/prefs"
{"error": 500}
```

Is there something I'm doing wrong here? Previously the API responded fine with not specifying any field name; it would return every preference.

See also [this thread](https://www.reddit.com/r/redditdev/comments/4118t8).

**Comment by 13steinj** [comment]
@thatJavaNerd I can't check, but I think @MelissaCole fixed this in 0bbc1e4

**Comment by thatJavaNerd** [comment]
It's fixed now, thanks!
